src/pepr/uds-operator-config/templates/secret.yaml

```diff
@@ -8,5 +8,5 @@ metadata:
 type: Opaque
 data:
 {{- range $key, $value := .Values.operator }}
-  {{ $key }}: {{ $value | b64enc | quote }}
+  {{ $key }}: {{ $value | toString | b64enc | quote }}
```

Comment:
Calling out an alternative here that might be a simpler template if we wanted to go that route:
```yaml
stringData:
{{- range $key, $value := .Values.operator }}
  {{ $key }}: {{ $value | quote }}
```